dev.ares.ares.yaml

```diff
@@ -21,6 +27,9 @@ modules:
       - make -j $FLATPAK_BUILDER_N_JOBS -C desktop-ui install build=release hiro=gtk3
         local=false platform=linux prefix=$FLATPAK_DEST
       - install -D -m 644 -t $FLATPAK_DEST/share/metainfo $FLATPAK_ID.metainfo.xml
+      - cd thirdparty/librashader && cargo run -p librashader-build-script -- --profile optimized
+      - install -D thirdparty/librashader/target/optimized/librashader.so ${FLATPAK_DEST}/lib/librashader.so
+      - install -D thirdparty/librashader/pkg/librashader.pc ${FLATPAK_DEST}/lib/pkgconfig/librashader.pc
```

Comment:
Instead of using the vendored version of librashader, perhaps we could make it its own module in the Flatpak manifest in order to get better control over updates and dependencies. Vendoring is not a recommended practive in downstream distributions of software, hence the question.

Are you aware of differences between the vendored and corresponding released versions of librasharder?

Reply:
I'll deal with it, don't worry too much.